src/dropdown.cpp

```diff
@@ -87,6 +87,9 @@ struct DropdownWindow : Window {
 
 	Dimension items_dim{}; ///< Calculated cropped and padded dimension for the items widget.
 
+	static inline Dimension selected_item_dimension;
```

Comment:
I would prefer if these were not stored globally:
* `DropDownListItem`/`DropDownMover` should get a `OnClick` method, which returns a `button_index`.
* This keeps the pixel position stuff in a single class, instead of splitting it between `DropDownMover` and `HandleBadgeConfigurationDropDownClick`.
* The button can then be passed to `OnDropdownSelect` as third parameter.